### PR TITLE
Actions Page Recovery Mode Tag - Dev-50

### PR DESCRIPTION
**Pull request by chinins**
## Description

This is a simple issue that adds a "Recovery" tag element at the top of the actions page, when the colony is put into recovery mode.

**New stuff** ✨

* Added `Recovery` string via `defineMessages` & `useIntl` hook
* Added styles for `Recovery` tag

Resolves Dev-50

### CODE REVIEW

**Comment by rdig** [comment]
Few notes:
- You don't need to put the issue id in the title of your PR
- You _do_ need to add it at the end of your PR, with one of [linear's link keywors](https://docs.linear.app/GitHub-and-GitLab-fa4b88df484343e4989538f066c729f3), even if it's a draft PR _(eg: "Resoves DEV-50")_
- Assign yourself to the PR
- Add a label to your PR _(in your case that would be `feature`)_
- Don't need to update the PR's description while in draft mode

File: src/modules/dashboard/components/ActionsPage/ActionsPage.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.css:9]: Please don't use half pixel values

Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.css:9]: The text would look better "centered" vertically if the top and bottom padding would just be `4px`

```suggestion
  padding: 4px 8px;
```

Screenshot from 2020-12-18 16-03-36

File: src/modules/dashboard/components/ActionsPage/ActionsPage.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.css:12]: This has no reason to be an absolute element.

- Refactor `.container` to not include a top border
- create a `divider` element that has the solely purpose of displaying that diving line _(the border you just removed)_
- place your tag component inside the `.container` element, above the divider

File: src/modules/dashboard/components/ActionsPage/ActionsPage.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.css:15]: You can safely use `--radius-large` for this one, to keep things consistent

File: src/modules/dashboard/components/ActionsPage/ActionsPage.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.tsx:274]: Please use the  `FormattedMessage` component where possible, rather than the programatic `formatMessage`.

```suggestion
        <p className={styles.recoveryTag}>
          <FormattedMessage {...MSG.recoveryTag} />
        </p>
```

Comment (rdig) [reply src/modules/dashboard/components/ActionsPage/ActionsPage.tsx:274]: The programmatic hook is a legacy leftover in `react-intl` and is mostly used for stuff like giving a component the ability to support both strings and message descriptors as values.

See this as an example:
https://github.com/JoinColony/colonyDapp/blob/6a8ca7f612960c8c7788833202eceb3f5a709216/src/modules/core/components/Heading/Heading.tsx#L55-L65

**Review by rdig — CHANGES_REQUESTED** [review]
This still needs a bunch of work needed until it can be merged, but you're getting there :)

Most stuff that I've pointed out is related to things you must not have known, so this is mostly on me for not telling them to you, but hey... it's a learning process :)

Also, I noticed to basically copy-pasted the css specs from the Figma file. We don't really do that. We use Figma for visual guidance, but we decide how to actually implement the css declarations based on our component's needs, maintenability or otherwise "what makes sense". Figma specs are auto-generated, and they seem to be kind of crappy :P

That's all I have for now, let me know if you need help with this.

File: src/modules/dashboard/components/ActionsPage/ActionsPage.tsx
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.tsx:276]: I think this is old code right? It looks like the old tag before the improvement you did, the conditional should be wrapped around the div above this.

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
It's looking good besides the small thing I pointed out :ok_hand:

File: src/modules/dashboard/components/ActionsPage/ActionsPage.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.css:12]: I don't think we need a fixed width, as we have no strict requirement for it's width. 

The padding will take care of that anyway.

Imagine this some other way. At some point, internationalization will kick in and we'll have multiple languages. Longer works will break this.

Just remove this and it's all fine.

File: src/modules/dashboard/components/ActionsPage/ActionsPage.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsPage.css:42]: Usually, when I do dividers, I just go without a border, and just set a `background-color`, and a `height` of `1px`.

But this is just a side note, you can do it however you like it :)

**Review by rdig — APPROVED** [review]
Left a note that I think you should still address _(basically remove the `width`)_.

But I'm pre-approving this, so after you make that change, you can merge it.

File: src/modules/dashboard/components/ActionsPage/ActionsPage.tsx
Comment (rdig) [reply src/modules/dashboard/components/ActionsPage/ActionsPage.tsx:271]: Yes it is old code. Pull the new version, and then re-review it, as It's all good now.

**Pull request merged**
